### PR TITLE
DO-956 in footer "©2020 – Data Ocean. Всі права захищені."

### PR DESCRIPTION
**Pull request by Litsyshyn**
Added "All rights reserved" text to platform footer.

https://razom.atlassian.net/browse/DO-956

### CODE REVIEW

File: src/components/pages/auth/LoginPage.jsx [hunk 1]
```diff
@@ -106,6 +106,11 @@ const LoginPage = ({ match, history, location }) => {
               <Route404 />
             </Switch>
           </div>
+          <div className="my-auto mx-auto xl:ml-0 xl:bg-transparent px-5 sm:px-8 py-8 xl:p-0 xl:shadow-none w-full sm:w-3/4 lg:w-2/4 xl:w-auto">
+            <div className="intro-x text-white xl:text-gray-700 lg:ml-20 sm:ml-20">
+              <p>©2020 – Data Ocean. {t('allRightsReserved')}.</p>
+            </div>
+          </div>
```
Comment (roman-tiukh) [inline src/components/pages/auth/LoginPage.jsx:113]: image

File: src/components/pages/SystemPage.jsx [hunk 2]
```diff
@@ -75,6 +75,11 @@ const SystemPage = (props) => {
           <SystemRoutes {...props} />
         </div>
       </div>
+      <div className="flex">
+        <div className="pt-4 m-auto text-white">
+          <p>©2020 – Data Ocean. {t('allRightsReserved')}.</p>
+        </div>
+      </div>
```
Comment (roman-tiukh) [inline src/components/pages/SystemPage.jsx:82]: image

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
bad positioning

**Comment by Litsyshyn** [comment]
I hope it looks better )

File: src/components/pages/auth/LoginPage.jsx [hunk 3]
```diff
@@ -73,7 +73,7 @@ const LoginPage = ({ match, history, location }) => {
             {/* </div> */}
           </div>
         </div>
-        <div className="h-screen xl:h-auto flex py-5 xl:py-0 my-10 xl:my-0">
+        <div className="grid grid-rows-100 gap-90 h-screen xl:h-auto flex py-5 xl:py-0 my-10 xl:my-0 relative">
```
Comment (roman-tiukh) [inline src/components/pages/auth/LoginPage.jsx:76]:
```suggestion
        <div className="h-screen xl:h-auto flex flex-col py-5 xl:py-0 my-10 xl:my-0">
```

File: src/components/pages/auth/LoginPage.jsx [hunk 4]
```diff
@@ -106,6 +106,11 @@ const LoginPage = ({ match, history, location }) => {
               <Route404 />
             </Switch>
           </div>
+          <div className="mx-auto my-auto mb-8 xl:ml-20 xl:absolute bottom-0 left-0 ">
+            <div className="text-white xl:text-gray-700">
+              <p>©2020-2021 – Data Ocean. {t('allRightsReserved')}.</p>
+            </div>
+          </div>
```
Comment (roman-tiukh) [inline src/components/pages/auth/LoginPage.jsx:113]:
```suggestion
          <div className="mx-auto mb-8 xl:ml-20 text-white xl:text-gray-700">
            ©2020-2021 – Data Ocean. {t('allRightsReserved')}.
          </div>
```

File: src/components/pages/SystemPage.jsx [hunk 5]
```diff
@@ -75,6 +75,11 @@ const SystemPage = (props) => {
           <SystemRoutes {...props} />
         </div>
       </div>
+      <div className="flex">
+        <div className="pt-4 xl:pl-64 lg:pl-32 md:pl-24 mx-auto text-white">
+          <p>©2020-2021 – Data Ocean. {t('allRightsReserved')}.</p>
+        </div>
+      </div>
```
Comment (roman-tiukh) [inline src/components/pages/SystemPage.jsx:82]:
```suggestion
      <div className="flex justify-center items-center mt-3 xl:ml-64 lg:ml-32 md:ml-24 text-white">
        ©2020-2021 – Data Ocean. {t('allRightsReserved')}.
      </div>
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**